src/style.css

```
@@ -1,3 +1,68 @@
+*  {
+  box-sizing: border-box
+ }
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  width: 100%;
+  padding: 0 50px;
+}
+
+.logo_img {
+  display: flex;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 0;
```

sorry, it was my mistake 🤦‍♂️
there is should be 20px
```suggestion
  margin-left: 20px;
```